.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: node_js
 node_js:
-  - "0.12"
+- "0.12"
 branches:
   only:
   - v1.0
+deploy:
```

Comment:
This was when I was trying to setup Travis to deploy automatically to heroku. Should I rename the PR?